typing/typecore.ml

```diff
@@ -224,79 +218,75 @@ let case lhs rhs =
 
 type expected_mode =
   { position : apply_position;
-    escaping_context : escaping_context option;
+    escaping_context : Env.escaping_context option;
     mode : Value_mode.t;
     tuple_modes : Value_mode.t list;
     (* for t in tuple_modes, t <= regional_to_global mode *)
   }
 
 
 let mode_return mode =
-  let position = Tail in
-  let escaping_context = Some Return in
-  let tuple_modes = [] in
-  { position; escaping_context; mode; tuple_modes }
+  { position = Tail;
+    escaping_context = Some Return;
+    mode;
+    tuple_modes = []}
```

Comment:
I would have rather you'd left these as they were. I prefer the let style because it's much easier to add intermediate definitions later when you need them. It is also more uniform when either all or none of the fields are punned.